README.md

```diff
@@ -0,0 +1,607 @@
+# Tariff Interpreter
+
+___
+This Interpreter is a reverence implementation for computing a JSON formatted bike-sharing tariff model.
+
+## Contents
+
+___
+
+* [Why this interpreter](#why-this-tariff-schema)
+* [General Interpreter Operation](#general-interpreter-operation)
+* [Tariff JSON Schemas](#tariff-json-schema)
+
+## Why this Tariff-Schema
+
+___
+There are two main reasons why the creation of a new pricing model has begun:
+
+* our current style for represent price tariffs are functions. On the one hand, this has the disadvantage that it leads
+  to inconsistencies between the tariffs and, on the other hand, it is difficult to create and read a tariff.
+* The old model was not able to provide a transparent breakdown of costs incurred similar to a receipt.
+
+These points were tried to improve specifically.
+
+## General Interpreter Operation
+
+___
+The Interpreter requires a [Tariff JSON Object](#tariff-json-schema) and a rental period.
+
+## Tariff JSON-Schema
+
+___
+Basically, there are three types of tariffs:
+
+* [SlotBasedTariff:](#slotbasedtariff)
+
+This type of tariff can represent tariff that depend only on the duration of a rental.
```

Comment:
Formatting off. Probably check out https://dillinger.io/ or something similar and write in there.